events/signals.py

```diff
@@ -47,26 +46,26 @@ def user_created_notification(sender, instance, created, **kwargs):
         ]
         notification_type = NotificationType.USER_CREATED
         context = {"user": instance}
+
         if len(recipient_list) == 0:
             logger.warning(
                 "No recipients for notification type '%s'" % notification_type,
                 extra={"user": instance.username},
             )
             return
+
         try:
             rendered_notification = render_notification_template(
                 notification_type, context
             )
         except NotificationTemplateException as e:
             logger.error(e, exc_info=True)
             return
-        try:
-            send_mail(
-                rendered_notification["subject"],
-                rendered_notification["body"],
-                get_email_noreply_address(),
-                recipient_list,
-                html_message=rendered_notification["html_body"],
-            )
-        except SMTPException as e:
-            logger.error(e, exc_info=True, extra={"user": instance.username})
+
+        send_mail(
+            rendered_notification["subject"],
+            rendered_notification["body"],
+            get_email_noreply_address(),
+            recipient_list,
+            html_message=rendered_notification["html_body"],
+        )
```

Comment:
This `user_created_notification` is disabled, and should be removed altogether now (I got a confirmation on that). But I guess that can be done separately from this PR.

Reply:
I'll actually remove it in this PR, but in a separate commit.